ansible/tasks/stickypedia.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: "create stickypedia user"
+  user:
+    name: "stickypedia"
+    shell: "usr/sbin/nologin"
+    home: "/var/www/stickypedia"
+    system: true
+
+
+- name: "install prerequisites for gollum"
+  apt:
+    name:
+      - "ruby"
+      - "ruby-dev"
+      - "make"
+      - "zlib1g-dev"
+      - "libicu-dev"
+      - "build-essential"
+      - "git"
+      - "cmake"
+    state: "present"
+  become: true
+
+block:
+  - name: "clone git with Stickypedia content"
```

Comment:
This won't work, the repo is private and this needs a deploy key

Reply:
fix